.travis.yml

```diff
@@ -50,8 +50,14 @@ before_script:
 script:
   - |
     if [[ ! -z "$WP_VERSION" ]] ; then
+      # Run the build because otherwise there will be a bunch of warnings about
+      # failed `stat` calls from `filemtime()`.
       npm install || exit 1
       npm run build || exit 1
+      # Syntax check for php-pegjs parser
+      node bin/create-php-parser.js || exit 1
```

Comment:
I think not yet - instead, we need to make sure we run our existing parsing tests against the PHP parser as well.